content/actions/using-workflows/workflow-commands-for-github-actions.md

```diff
@@ -580,6 +580,8 @@ echo "{environment_variable_name}={value}" >> $GITHUB_ENV
 
 You can make an environment variable available to any subsequent steps in a workflow job by defining or updating the environment variable and writing this to the `GITHUB_ENV` environment file. The step that creates or updates the environment variable does not have access to the new value, but all subsequent steps in a job will have access. The names of environment variables are case-sensitive, and you can include punctuation. For more information, see "[Environment variables](/actions/learn-github-actions/environment-variables)."
 
+{% data reusables.actions.environment-variables-are-fixed %} For more information about the default environment variables, see "[AUTOTITLE](/actions/learn-github-actions/environment-variables#default-environment-variables)."
+
 ### Example
```

Comment:
The current text of this subheading is:
```suggestion
### Example of writing an environment variable to `GITHUB_ENV`
```

Changing this will avoid a conflict.